### PR TITLE
Disable coveralls to speed up ci.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,7 +34,6 @@ jobs:
          - cargo install cargo-kcov
          - kcov --version
```
Comment (rzumer) [inline .travis.yml]: What about these two lines?

Comment (lu-zero) [reply .travis.yml]: They should go as well as the kcov installation above.

**Comment by lu-zero** [comment]
We can do w/out kcov, hopefully something like https://github.com/xd009642/tarpaulin will get ready for use later in the year.

**Comment by EwoutH** [comment]
Maybe we could do this a little more refined? Like only running Coveralls when merging into master?

**Comment by lu-zero** [comment]
I'd give tarpaulin a go today, if it works could you please prepare a nightly-only task @EwoutH ?

**Comment by tdaede** [comment]
Closing in favor of tarpaulin.

**Pull request closed without merging**